src/action-controller/base.cr

```diff
@@ -4,9 +4,11 @@ require "./responders"
 require "./session"
 require "./support"
 require "uri"
+require "../../spec/curl_context"
 
 abstract class ActionController::Base
   include ActionController::Responders
+  include ActionController::Context
```

Comment:
It would be better to do this in `curl_context`. That way it doesn't pollute the base class if you're not using it, as `ActionController::Context` provides no functionality outside of writing specs

Reply:
Wasn't was what I did the same as what you mean by your previous comment ?
`What do you think about reopening ActionController::Controller to include these methods? That way you don't need to instantiate the module with the type, and can use the type directly.`

Reply:
@dukeraphaelng I think what he means is that you should re-open `ActionController::Base` in `curl_context.cr`
i.e.

```
# curl_context.cr

abstract class ActionController::Base
    include ActionController::Context
end
```

We don't want to pollute the base class outside of running the `specs`

Reply:
Ok I see what you mean now